lighttpd/config.sls

```diff
@@ -2,14 +2,28 @@
 #
 # Manages the main lighttpd server configuration file.
 
-{% from "lighttpd/map.jinja" import lighttpd with context %}
-{% from "lighttpd/macros.jinja" import files_switch with context -%}
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_package_install = tplroot ~ '.install' %}
+{% from tplroot ~ "/map.jinja" import lighttpd with context %}
+{% from tplroot ~ "/macros.jinja" import files_switch with context -%}
 {% set file = lighttpd.get('config', ['lighttpd.conf']) %}
 
+include:
+  - {{ sls_package_install }}
 
 lighttpd_config:
   file.managed:
     - name: /etc/lighttpd/{{ lighttpd.config }}
-    - source: {{ files_switch('lighttpd', ['/etc/lighttpd/' ~ file,
-                                           '/etc/lighttpd/' ~ file ~ '.jinja']) }}
+    - source: {{ files_switch(
+                      salt['config.get'](
+                        tplroot ~ ':tofs:files:lighttpd_config',
+                        ['/etc/lighttpd/lighttpd.conf', '/etc/lighttpd/lighttpd.conf.jinja']
```

Comment:
This is the default template.  Since you're only using `/etc/lighttpd/lighttpd.conf` in the repo, `/etc/lighttpd/lighttpd.conf.jinja` can be removed.